src/auth/auth-api-request.ts

```diff
@@ -597,48 +596,41 @@ export class FirebaseAuthRequestHandler {
    * Invokes the request handler based on the API settings object passed.
    *
    * @param {ApiSettings} apiSettings The API endpoint settings to apply to request and response.
-   * @param {Object} requestData The request data.
-   * @return {Promise<Object>} A promise that resolves with the response.
+   * @param {object} requestData The request data.
+   * @return {Promise<object>} A promise that resolves with the response.
    */
-  private invokeRequestHandler(apiSettings: ApiSettings, requestData: Object): Promise<Object> {
-    let path: string = this.path + apiSettings.getEndpoint();
-    let httpMethod: HttpMethod = apiSettings.getHttpMethod();
+  private invokeRequestHandler(apiSettings: ApiSettings, requestData: object): Promise<object> {
+    const path: string = this.path + apiSettings.getEndpoint();
+    const httpMethod: HttpMethod = apiSettings.getHttpMethod();
     return Promise.resolve()
       .then(() => {
         // Validate request.
-        let requestValidator = apiSettings.getRequestValidator();
+        const requestValidator = apiSettings.getRequestValidator();
         requestValidator(requestData);
         // Process request.
         return this.signedApiRequestHandler.sendRequest(
             this.host, this.port, path, httpMethod, requestData, this.headers, this.timeout);
       })
       .then((response) => {
         // Check for backend errors in the response.
-        let errorCode = FirebaseAuthRequestHandler.getErrorCode(response);
+        const errorCode = FirebaseAuthRequestHandler.getErrorCode(response);
         if (errorCode) {
           throw FirebaseAuthError.fromServerError(errorCode, /* message */ undefined, response);
         }
         // Validate response.
-        let responseValidator = apiSettings.getResponseValidator();
+        const responseValidator = apiSettings.getResponseValidator();
         responseValidator(response);
         // Return entire response.
         return response;
       })
       .catch((response) => {
-        let error;
-        if (typeof response === 'object' && 'statusCode' in response) {
-          // response came directly from a non-200 response.
-          error = response.error;
-        } else {
-          // response came from a thrown error on a 200 response.
-          error = response;
-        }
-
+        const error = (typeof response === 'object' && 'statusCode' in response)
```

Comment:
Better for readability to have the ? on the previous line:
```
const error = (typeof response === 'object' && 'statusCode' in response) ?
  response.error : response;
```
Here and elsewhere

Reply:
Done